core/src/OpenACC/Kokkos_OpenACC_ParallelFor_MDRange.hpp

```diff
@@ -114,7 +114,8 @@ struct Kokkos::Experimental::Impl::OpenACCParallelForHelper<
         }
       }
     } else {
-      static_assert("implementation bug");
+      static_assert(
+          "Kokkos Error: not supported memory layout for the OpenACC backend");
```

Comment:
No, I really meant that comment.  If this else-clause gets compiled there is a bug, it is not a matter of being supported or not.

Reply:
Updated the comment.